components/Grid/Wrappers/GridWrapperHeaderCell.vue

```diff
@@ -90,3 +91,11 @@ export default {
     },
 };
 </script>
+
+<style lang="scss" scoped>
+    .fixed-header {
+        position: sticky !important;
```

Comment:
this `important` is necessary ?

Reply:
Yes, otherwise it will be covered by position: relative